sequence/src/main/java/com/vitreoussoftware/bioinformatics/sequence/InvalidDnaFormatException.java

```diff
@@ -6,6 +6,7 @@
  *
  */
 public class InvalidDnaFormatException extends RuntimeException {
+	private static final long serialVersionUID = 1L;
```

Comment:
So, another option is @SuppressWarnings("serial") in which case it will auto-generate a serial number (although this may apparently have issues between compilers/systems/etc.).  Of course, the auto-generated one would definitely change if you changed the class, which a fixed value one would not. Of course, if the object isn't going to be serialized ever, then it doesn't really matter either way and I find the annotation cleaner, but anyway.